src/OpenLoco/Industry.cpp

```diff
@@ -82,7 +82,7 @@ namespace OpenLoco
     {
         char* ptr = (char*)buffer;
         *ptr = '\0';
-        auto industryObj = object();
+        auto industryObj = getObject();
```

Comment:
```suggestion
        const auto* industryObj = getObject();
```
Makes it a bit clearer imho.